File: Javascript/UberBlack.js
class UberBlack extends Car {
    constructor(license, driver, brand, typeCar_Accepted, seats) {
        super(license, driver);
        this.brand = brand;
        this.typeCar_Accepted = typeCar_Accepted;
        this.seats = seats;
    }
    printDataCar() {
        super.printDataCar();
        console.log(this.brand);
    }
}

File: Python/UberBlack.py
from car import Car


class UberBlack(Car):
    typeCar_Accepted = []
    seats_Material = []

    def __int__(self, license, driver, typeCar_Accepted, seats_Material):
        super().__init__(license, driver)
        self.typeCar_Accepted = typeCar_Accepted
        self.seats_Material = seats_Material

